### PR TITLE
Distributed viewport update

### PR DESCRIPTION
**Pull request by devinaconley**
## Overview
This updates the distributed example to configure camera viewports with the new method

## Changes
- configure a `Camera` sensor with a nested `Viewport` object
- modify camera position on vehicle
- bug fix to not wait for sensors that aren't streaming

## Testing
Launch simulators on ports 8999, 9000, 9001

Run the distributed example

Should see two viewports open up displaying a front right and front left camera view

### CODE REVIEW

File: examples/cpp/distributed/replica_lidar_sensors.json [hunk 1]
```diff
@@ -45,6 +46,12 @@
     "min_shutter": 0.000500,
     "max_shutter": 0.001400,
     "sensor_size": 9.07,
-    "channels": "bgra"
+    "channels": "bgra",
+    "viewport": {
```
Comment (bradleynull) [inline examples/cpp/distributed/replica_lidar_sensors.json:50]: We need an issue in the documentation repo to update all the camera configuration examples after this is merged.

**Review by bradleynull — CHANGES_REQUESTED** [review]
@devinaconley This is working for me. I was curious if we could make the example start the viewports in "windowed" mode so we have a window border and title? Mine just pop up and are not movable: 
image

Other than that, just one other minor comment.

File: examples/cpp/distributed/replica_lidar_sensors.json (hunk 1 above)
Comment (devinaconley) [reply examples/cpp/distributed/replica_lidar_sensors.json:50]: Yep I have an issue for myself on this one

**Comment by devinaconley** [comment]
@bradleynull - pushed the windows slightly off the top border, should be able to move them now!

**Review by bradleynull — APPROVED** [review]
@devinaconley Looks good to go now. Approved!

**Pull request merged**